docs/beyond_basics/101-164-dataladdening.rst

```diff
@@ -0,0 +1,141 @@
+Consider :ref:`applying procedures <procedures>` with ``-c <procedure-name>`` do apply configurations that suit your use case.
+
+.. findoutmore:: What is my directory is already a Git repository?
+
+   If you want to transform a Git repository to a DataLad dataset, a :command:`datalad create -f` is the way to go, too, and completely safe.
+   Your Git history will stay intact and will not be tempered with.
+
+If you want to transform a series of nested directories into nested datasets, continue with :command:`datalad create -f` commands in all further subdirectories.
+
+.. findoutmore:: One or many datasets?
+
+   In deciding how many datasets you need, try to follow the benchmarks in chapter :ref:`chapter_gobig` and the yoda principles in section :ref:`yoda`.
+   Two simple questions can help you make a decision:
+
+   #. Do you have independently reusable components in your directory, for example data from several studies, or data and code/results? If yes, make each individual component a dataset.
+   #. How large is each individual component? If it exceeds 100-200k files, split it up into smaller datasets. The decision on where to place subdataset boundaries can be guided by the existing directory structure or by common access patterns, for example based on data type (raw, processed, ...) or subject association. One straightforward organization may be a top-level superdataset and subject-specific subdatasets, mimicking the structure chosen in the use case :ref:`usecase_HCP_dataset`.
+
+You can automate this with :term:`bash` loops, if you want.
+
+.. findoutmore:: Example bash loops
+
+   Consider a directory structure that follows a naming standard such as `BIDS <https://bids.neuroimaging.io/>`_::
+
+      # create a mock-directory structure:
+      $ mkdir -p study/sub-0{1,2,3,4,5}/{anat,func}
+      $ tree study
+      study
+        ├── sub-01
+        │   ├── anat
+        │   └── func
+        ├── sub-02
+        │   ├── anat
+        │   └── func
+        ├── sub-03
+        │   ├── anat
+        │   └── func
+        ├── sub-04
+        │   ├── anat
+        │   └── func
+        └── sub-05
+            ├── anat
+            └── func
+
+   Consider further that you want to transform all ``sub-*`` directories into individual datasets.
+   Here is a line that would do this for the example above::
+
+      $ for dir in study/sub-0{1,2,3,4,5}; do datalad -C $dir create --force; done
+
+Step 3: Saving dataset contents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any existing content in your newly created dataset(s) still needs to be saved into its dataset at this point (unless it was already under version control with Git).
+This can be done with the :command:`datalad save` command -- either "in one go" using a plain ``datalad save`` (saves all untracked files and modifications to a dataset -- by default into the dataset annex), or step-by-step by attaching paths to the ``save`` command.
+Make sure to run :command:`datalad status` frequently.
```

Comment:
If done as suggested above, there should be no need to explore via `datalad status`.